src/include/fe_utils/frontend_hook.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#ifndef FRONTEND
+#define FRONTEND
+#endif
+
+extern char pg_install_path[], pg_data_path[];
```

Comment:
let's have a prefix in the var to avoid name collisions. `pg_install_path` looks quite common as a global var.